.golangci.yaml

```diff
@@ -26,4 +27,12 @@ linters:
     # - gocyclo
     # - prealloc
     # - maligned
-    # - gosec
+
+linters-settings:
+  gosec:
```

Comment:
Here is the list of current gosec rules that seem interesting based on the short description:

```
    G103: Audit the use of unsafe block
    G104: Audit errors not checked
    G109: Potential Integer overflow made by strconv.Atoi result conversion to int16/32
    G110: Potential DoS vulnerability via decompression bomb
    G204: Audit use of command execution
    G301: Poor file permissions used when creating a directory
    G302: Poor file permissions used with chmod
    G303: Creating tempfile using a predictable path
    G304: File path provided as taint input
    G305: File traversal when extracting zip/tar archive
    G306: Poor file permissions used when writing to a new file
```

* Is there a way to read up on those rules? I can't find explanations on the website or the repo.
* Why is `G204` disabled? Seems like the most useful one?
* Why is `G601: Implicit memory aliasing of items from a range statement` a security bug? Why is it enabled?
* Why is `0660` better than `0666`? Assuming that umask is sensible.